Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -36,6 +55,13 @@ configuration:
   - Incoming
   - Outgoing
   - Both
+- defaultvalue: '1'
+  display: XDR mirroring delay in minutes
+  name: xdr_delay
```

Comment:
Maybe we should think of less intimidating name for this parameter - maybe mirroring window lookback or something like that